packages/utils/src/bulk-trigger-patient-discovery.ts

```diff
@@ -0,0 +1,81 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import { out } from "@metriport/core/util/log";
+import { sleep } from "@metriport/shared";
+import axios from "axios";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { chunk } from "lodash";
+
+/**
+ * This script kicks off the patient discovery for the indicated patients.
+ *
+ * To run:
+ * 1. Set the env vars:
+ *  -CX_ID
+ *  -API_URL
+ * 2. Set the patientIds and rerunPdOnNewDemographics
+ * 3. Run the script with `ts-node src/bulk-trigger-patient-discovery.ts`
+ */
+
+dayjs.extend(duration);
+
+const confirmationTime = dayjs.duration(10, "seconds");
+const delayTime = confirmationTime.asMilliseconds();
+const cxId = getEnvVarOrFail("CX_ID");
+const apiUrl = getEnvVarOrFail("API_URL");
+const CHUNK_DELAY_MAX_MS = dayjs.duration({ seconds: 1 }).asMilliseconds();
+const PATIENT_CHUNK_SIZE = 5;
+
+const patientIds: string[] = [];
+const rerunPdOnNewDemographics = true;
+
+type PdResponse = {
+  data: {
+    requestId: string;
+  };
+};
+
+async function main() {
+  try {
+    await displayInitialWarningAndConfirmation(patientIds.length);
+
+    const patientChunks = chunk(patientIds, PATIENT_CHUNK_SIZE);
+
+    for (const [j, patients] of patientChunks.entries()) {
```

Comment:
Why j instead of i?

Reply:
took this part from the inner loop of a different script and didnt change the letter haha